### ISSUE TITLE
[COGNITION]: Direct Deposit - Consider revising the Call to Action UI for entering data

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1331]
## Description

There are three calls to action that all open the same form for entering direct deposit data. This could be potentially confusing. I'd like to review this with design and content. Happy to close if this is a non-issue. Screenshot attached below.

/cc @emilywaggoner @peggygannon 

## Point of Contact

**VFS Point of Contact:** _Trevor, Emily, Peggy_

## Acceptance Criteria

* As a user, I would like to have a clear call to action for entering my direct deposit data

## Screenshots or Trace Logs

![Screen Shot 2019-08-27 at 3.56.18 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/3bf6e0a9-c459-477b-9578-b95780646a9c)

**Comment by peggygannon** [comment]
@emilywaggoner @1Copenut L 

I wasn't part of team that initially created this page, so I'm not sure why we list 3 separate links to go to same form.  I agree having one button or link would be better. The whole page uses links so I would suggest tweaking the copy and having a link (as opposed to a button) for consistent formatting.  I can upload some suggestions for copy tweaks to support this.

Let me know if you want to get on a call to discuss.

**Comment by 1Copenut** [comment]
@peggygannon @emilywaggoner 
Visually I think we can make the trigger appear as a link or button, I'll defer to you both for that distinction. I feel like it should be a `<button>` element in the HTML since we're opening a modal dialog instead of navigating users to a new page.

Around all day if you'd like to move this to a call or video chat.

**Comment by Justin-Pickett** [comment]
will revisit this as a post launch direct deposit item.

**Comment by Justin-Pickett** [comment]
This item will be revisited in Personalization 2.0 combine profile and account work. Will close this ticket.

**Comment by 1Copenut** [comment]
@Justin-Pickett Is this on a roadmap for work in the new year? I'm revising the product 508 list and asking about open items.

**Comment by 1Copenut** [comment]
@Justin-Pickett Is this still on the roadmap for 2020 work?
